src/core/currency/wallet/currency-wallet-pixie.ts

```diff
@@ -249,12 +254,13 @@ export const walletPixie: TamePixie<CurrencyWalletProps> = combinePixies({
             startupPromise
               .then(() => engine.killEngine())
               .catch(error => input.props.onError(error))
-              .then(() =>
+              .then(() => {
+                input.onOutput({ ...walletOutput, engineStarted: false })
```

Comment:
This would also need to be fixed if my previous comment turns out to be correct.